.vscode/launch.json

```diff
@@ -6,13 +6,7 @@
       "request": "launch",
       "name": "Jest Current File",
       "program": "${workspaceFolder}/node_modules/.bin/jest",
-      "args": [
-        "${file}",
-        "--config",
-        "jest.config.js",
-        "--no-cache",
-        "--testTimeout=2147483647"
```

Comment:
Sorry @Andarist, I think this was a bad idea. With this test timeout, any failing test will now hang for a long time instead of timing out, making it very difficult to determine which test is failing.

Reply:
Let's revert this change from here. I opened a PR [here](https://github.com/statelyai/xstate/pull/4024) to add a second launch script - there will be one with the timeout and one without it.